kythe/java/com/google/devtools/kythe/analyzers/java/JavaEntrySets.java

```diff
@@ -89,7 +89,7 @@ public VName getNode(
       Symbol sym,
       String signature,
       // TODO(schroederc): separate MarkedSource generation from JavaEntrySets
-      @Nullable MarkedSource.Builder msBuilder,
```

Comment:
JFYI, this introduces a google-java-format issue with our presubmits, have you set up the commit hooks as documented at https://www.kythe.io/contributing/#linting-and-testing ?